tests/testthat/test_estimator.R

```diff
@@ -22,30 +22,30 @@ test_that("create, submit experiment, run in default amlcompute,
                    compute_target = existing_compute$name, 
                    entry_script = script_name, 
                    script_params = list("data_folder" = ds$as_mount()),
-                   cran_packages = c("ggplot2", "dplyr"))
+                   cran_packages = c("dplyr", "ggplot2"))
   
   run <- submit_experiment(exp, est)
   wait_for_run_completion(run, show_output = TRUE)
   
   run <- get_run(exp, run$id)
   metrics <- get_run_metrics(run)
   
-  expected_metrics <- list("test_metric" = 0.5)
-  expect_equal(length(setdiff(metrics, expected_metrics)), 0)
+  expect_equal(metrics$test_metric, 0.5)
   
-  # Uncomment once base image upgrades the sdk
-  # expected_list <- c(1, 2, 3)
-  # expect_equal(length(setdiff(metrics$test_list, expected_list)), 0)
+  expected_list <- c(1, 2, 3)
+  expect_equal(length(setdiff(metrics$test_list, expected_list)), 0)
 
-  # expected_row <- list(x = 1, y = 2)
-  # expect_equal(length(setdiff(metrics$test_row, expected_row)), 0)
+  expected_row <- list(x = 1, y = 2)
+  expect_equal(length(setdiff(metrics$test_row, expected_row)), 0)
 
-  # expect(startsWith(metrics$test_predictions, "aml://artifactId") &&
-  #       endsWith(metrics$test_predictions, "test_predictions"),
-  #       "invalid predictions uri returned")
+  expect(startsWith(metrics$test_predictions, "aml://artifactId") &&
+         endsWith(metrics$test_predictions, "test_predictions"),
+         "invalid predictions uri returned")
+  
+  upload_files_to_run(run, list("dummy_data"), list("dummy_data.txt"))
```

Comment:
can you add test for upload_folder_to_run?

Reply:
Added.